src/socket-types.ts

```diff
@@ -206,6 +206,40 @@ export enum ChainType {
   default = "default",
 }
 
+export enum FinalityBucket {
+  fast = 1,
+  medium = 2,
+  slow = 3,
+}
+
+export enum TransmissionParamFinalityType {
+  bucket = 1,
+  block = 2,
+  time = 3,
+}
+
+export enum BucketFinalityType {
+  block = 1,
+  time = 2,
+}
+
+export type ChainFinalityInfo = {
+  type: BucketFinalityType;
+  defaultBucket: FinalityBucket;
```

Comment:
do we need defaultBucket in ChainFinalityInfo? can we simply default to fast or medium ?

Reply:
i am using slow as default Bucket for ethereum and polygon with 18 and 256 confirmations respectively. someone might want to relay a message faster on these chains and can use fast. so feel like should keep it for such configurations

Reply:
we need better definition for buckets, I will revisit this comment